src/main/java/com/nextstep/lotto/LottoSeller.java

```diff
@@ -0,0 +1,41 @@
+package com.nextstep.lotto;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class LottoSeller {
+    public static final int LOTTO_PRICE = 1000;
+    private static final int LOTTO_START_IDX = 0;
+    private static final int LOTTO_NUMBER_COUNT = 6;
+    private static final int LOTTO_MAX_COUNT = 45;
+
+    private LottoSeller(){}
+
+    public static int count(int price) {
+        return price / LOTTO_PRICE;
+    }
+
+    public static Lottos buy(int count) {
+        Lottos lottos = new Lottos();
```

Comment:
```suggestion
        return IntStream.range(0, count)
                .mapToObj(i -> new Lotto(getRandomNumbers()))
                .collect(Collectors.collectingAndThen(Collectors.toList(), Lottos::new));
```
이렇게 하면 lottos의 add 메서드를 없앨 수 있겠네요